app/api/search/search.js

```diff
@@ -18,7 +18,34 @@ import templatesModel from '../templates';
 import { bulkIndex, indexEntities, updateMapping } from './entitiesIndex';
 import thesauri from '../thesauri';
 
-function processFilters(filters, properties) {
+function processParentThesauri(property, values, dictionaries, properties) {
+  if (!values) {
+    return values;
+  }
+
+  let sourceProperty = property;
```

Comment:
can be changed, if wanted to:

  const sourceProperty =
    property.type === 'relationship'
      ? properties.find(p => p._id.toString() === property.inherit.property.toString())
      : property;

not necessary though

Reply:
It can, indeed. Will take a look!

Reply:
Great :)